java/src/main/java/com/amido/stacks/workloads/menu/api/v2/MenuControllerV2.java

```diff
@@ -0,0 +1,57 @@
+package com.amido.stacks.workloads.menu.api.v2;
+
+import com.amido.stacks.core.api.annotations.ReadAPIResponses;
+import com.amido.stacks.workloads.menu.api.v1.dto.response.MenuDTO;
+import com.amido.stacks.workloads.menu.commands.MenuCommand;
+import com.amido.stacks.workloads.menu.domain.Menu;
+import com.amido.stacks.workloads.menu.exception.MenuNotFoundException;
+import com.amido.stacks.workloads.menu.mappers.MenuMapper;
+import com.amido.stacks.workloads.menu.service.MenuQueryService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequestMapping(path = "/v2/menu", produces = "application/json; charset=utf-8")
+@RestController
+public class MenuControllerV2 {
+
+  @Autowired private MenuMapper menuMapper;
```

Comment:
Possibly convert to use Constructor injection, if that's the pattern we have decided to follow?

Reply:
this was my question ?!

Reply:
Yes, let's go for CI :-)